Makefile

```diff
@@ -4,13 +4,14 @@ DIST          = $(NAME)-$(VERSION)/
 
 all: test build build-so
 
-dist: clean internal/connect/version.txt
+dist: download_deps clean internal/connect/version.txt
```

Comment:
Is this really needed? Could you explain why?

Reply:
This was a redundancy but will be cleaned up later.

Reply:
Why not cleaning it now?